QuizApp/Presenters/LogInPresenter.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  LogInPresenter.swift
+//  QuizApp
+//
+//  Created by Lovre on 14/05/2021.
+//
+
+import Foundation
+
+protocol LogInProtocol{
+    func login(email: String, password: String) -> Void
+}
+
+class LogInPresenter:LogInProtocol{
+    private let router:AppRouter!
+    
+    init(router:AppRouter){
+        self.router = router
+    }
+    
+    func login(email: String, password: String) -> Void {
+        
+        let networkService = NetworkService<Userinfo>()
```

Comment:
Network service se treba kreirati samo jednom. Na ovaj način svaki puta kada user pritisne login gumb, ti kreiraš novu instancu NetworkServicea.